### PR TITLE
Display readable errors instead of html

### LINKED ISSUE #36: Upload errors should be displayed better
Currently, errors that occur during an upload will spew a bunch of html in the upload box. We should figure out a better way to render this information (modal? error message at the top bar?)

### PR DESCRIPTION
**Pull request by sea-kelp**
## Description of Changes
Fixes #36.

Displays readable upload errors instead of spewing html.

* Added json error handlers for requests with "Accept: application/json" headers
* Added a default error message if the server still sends an html error response

## Notes for Deployment
None!

## Screenshots (if appropriate)
Errors shown when upload failed due to:
- internal server error
- file too large
- file type not allowed

err1

When an html error is received, the caption is set to "Upload failed." but if you hover over it you still get a bunch of html. This seems to be set in the dropzone library and there's not much we can do about it

err2

## Tests and linting

 - [x] I have rebased my changes on `main`

 - [x] `just lint` passes

 - [x] `just test` passes

### CODE REVIEW

File: OpenOversight/app/templates/submit_image.html [hunk 1]
```diff
@@ -76,12 +76,20 @@ <h3>Drop images here to submit photos of officers:</h3>
             acceptedFiles: "image/png, image/jpeg, image/gif, image/jpg, image/webp",
             maxFiles: 50,
             headers: {
+                'Accept': 'application/json',
```
Comment (AetherUnbound) [inline OpenOversight/app/templates/submit_image.html:79]: This looks duplicates on the submit officer image page - is there any way these can be unified or do they differ enough to warrant separation?

**Review by AetherUnbound — COMMENTED** [review]
Seriously amazing job!! This is SO much better than what we were seeing before. And so simple! Just a couple comments/questions

File: OpenOversight/app/templates/submit_image.html (hunk 1 above)
Comment (sea-kelp) [reply OpenOversight/app/templates/submit_image.html]: Good point, I've moved them to a new file

**Review by AetherUnbound — APPROVED** [review]
Excellent work :100:

**Pull request merged**